src/Serializer/ItemNormalizer.php

```diff
@@ -30,8 +30,6 @@
 /**
  * Generic item normalizer.
  *
- * @final
```

Comment:
I don't think this is right. The GraphQL ItemNormalizer should stop extending this class instead.